source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1557,8 +1560,9 @@ def _getIA2RelationFirstTarget(
 						IAccessibleObject=ia2Object,
 						IAccessibleChildID=0
 					)
-			except COMError:
-				pass
+		except (NotImplementedError, COMError):
+			log.debug("Unable to fetch _IA2Relations")

```

Comment:
As this exception surrounds  code where it is likely the exception may be raised several frames deep, I think we should log the traceback.
 ```suggestion
			log.debug("Unable to fetch _IA2Relations", exc_info=True)
```

Reply:
Yeah, ok. I've added this. Note that exception paths in `self._IA2Relations:` all have explicit debug logging. There are other locations that may raise, so I think the stack will be helpful, but may be a bit noisy (occasional log messages using firefox) when logging at debug level.